browser-api/src/main/java/com/duckduckgo/app/notification/NotificationSender.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.notification
+
+import com.duckduckgo.app.notification.model.SchedulableNotification
+
+interface NotificationSender {
```

Comment:
Created in the browser-api to be able to use it in the macos module

Reply:
Let's plan to create a notification module in the future